server/src/test/java/org/elasticsearch/action/search/SearchResponseTests.java

```diff
@@ -116,12 +99,11 @@ private SearchResponse createTestItem(boolean minimal, ShardSearchFailure... sha
         if (minimal == false) {
             SearchHits hits = SearchHitsTests.createTestItem(true, true);
             try {
-                InternalAggregations aggregations = aggregationsTests.createTestInstance();
```

Comment:
This test looks obsolete as it is testing we can read and write search responses to xcontent which is not needed any more.